service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/KubernetesClientProviderImpl.java

```diff
@@ -40,7 +40,7 @@ private KubeConfig loadKubeConfig(
             .manager()
             .serviceClient()
             .getManagedClusters()
-            .listClusterUserCredentials(mrgName, aksClusterName)
+            .listClusterAdminCredentials(mrgName, aksClusterName)
```

Comment:
Why this change is required?

Reply:
We are switching to admin credentials to avoid Terra services having to use the aad auth. We don't use the kubernetes rbac, so there's isn't a good reason to user credentials over admin credentials.